src/utils/cql/evaluateCql.js

```diff
@@ -0,0 +1,65 @@
+const { Client } = require('cql-translation-service-client');
+const { Library, Executor, Repository } = require('cql-execution');
+const { PatientSource } = require('cql-exec-fhir');
+const { Base64 } = require('js-base64');
```

Comment:
This is pretty minor but I noticed it right away - rather than add a new dependency, is there any reason we can't use `btoa` / `atob` here?